evm/bridge-token-factory/contracts/BridgeTokenFactory.sol

```diff
@@ -36,6 +36,7 @@ contract BridgeTokenFactory is
 
     address public tokenImplementationAddress;
     address public nearBridgeDerivedAddress;
+    uint8 private _omniBridgeChainId;
```

Comment:
Is there are any benefits of making it private? If not, then let's make it public

Reply:
Just couldn't see a reason anyone would need this and want to keep API smaller

Reply:
4887048bd4a026929e22219c733a0e5921faf3cc